vertx-platform/src/main/java/org/vertx/java/platform/impl/cli/Starter.java

```
@@ -269,6 +271,15 @@ private void runBareHA(Args args) {
     block();
   }
 
+  private ConfigurationLoader getConfigurationLoader(String className) {
+    try {
+      return (ConfigurationLoader) Class.forName(className).newInstance();
+    } catch (InstantiationException | IllegalAccessException | ClassNotFoundException e) {
+      log.error("Could not instantiate configuration loader: " + className, e);
+      return null;
```

And I would throw this exception, so we do not swallow the exception